vendor-extra/LiberoPageBundle/src/EventListener/TitleListener.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPageBundle\EventListener;
+
+use Libero\LiberoPageBundle\Event\CreatePageEvent;
+use Libero\ViewsBundle\Views\TranslatingVisitor;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use function is_string;
+
+final class TitleListener
+{
+    use TranslatingVisitor;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function onCreatePage(CreatePageEvent $event) : void
+    {
+        $title = $event->getTitle();
+
+        if (is_string($title)) {
+            $title .= ' | ';
```

Comment:
Not sure if needs localisation?

Reply:
It's been a while since I have worked on a site with multiple languages but we will need instances of translatable text which accept variables such as `Welcome back, [username]`. Can we have a translatable string for the site title when page and site title is available: `[page_title] | [site_title]`.

Reply:
👍